WordPress/src/main/java/org/wordpress/android/ui/accounts/NewUserFragment.java

```diff
@@ -488,8 +488,12 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         termsOfServiceTextView.setOnClickListener(new OnClickListener() {
                                                       @Override
                                                       public void onClick(View v) {
+                                                          String localString =  LanguageUtils.getPatchedCurrentDeviceLanguage(v.getContext());
+                                                          if (localString.contains("_")) {
```

Comment:
This is necessary for transforming `fr_fr` to just `fr`, but `pt_br` should be changed to `pt-br`.